docs/source/markdown/podman-search.1.md

```diff
@@ -65,6 +68,11 @@ Example if limit is 10 and two registries are being searched, the total
 number of results will be 20, 10 from each (if there are at least 10 matches in each).
 The order of the search results is the order in which the API endpoint returns the results.
 
+**--list-tags**
+
+List the available tags in the repository for each image found. Note: --list-tags requires the search term to be a full
+repository name. If not specified the registry server, default registries will be searched through (in /etc/containers/registries.conf).  The result contains the Image name and its tag, one line for every tag associated with the image.
+
```

Comment:
This is all very contradictory.  You're saying at the start that you'll list all tags for each image found in the repository, and then that the full name must be used.  I think from our conversations that the image name with the repository must be supplied.  I.e.  you must do `docker.io/library/alpine` and you can't do `docker.io/library/` or `alpine`.  Given that assumption:
```suggestion
List the available tags in the repository for the specified image. **Note:** --list-tags requires the search term to be a fully
specified image name. The result contains the Image name and its tag, one line for every tag associated with the image.
```

Reply:
fixed

Reply:
I think the docker.io/library/alpine refer to a repository name, if append a tag to it, it will refer to an image?

Reply:
@QiWang19 yeah, it's confusing and I may not have helped matters.  When you push a container image to a registry, it creates a repository from it.  How's this?

```
List the available tags in the repository for the specified image. **Note:** --list-tags requires the search term to be a fully
specified image. The result contains the Image name and its tag, one line for every tag found in the repository.
```

Reply:
:+1: 
nicely done @QiWang19